benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -613,9 +615,9 @@ int main(int argc, char *argv[])
 #endif
 
 #ifdef KOKKOS_ENABLE_SERIAL
-    if (spec.backends == "all" || spec.backends == "rtree")
-      register_benchmark<Kokkos::Serial, BoostExt::RTree<ArborX::Point>>(
-          "BoostRTree", spec);
+      // if (spec.backends == "all" || spec.backends == "rtree")
+      // register_benchmark<Kokkos::Serial, BoostExt::RTree<ArborX::Point>>(
+      // "BoostRTree", spec);
 #endif
```

Comment:
This is due to `BoostRTree` is not implementing `bounds()`. Right now, we have a tighter than wanted connection between `query_crs` and `query()` through sorting routine. The sorting right now is done using Morton codes, which requires `bounds()`.